content/kubeone/master/architecture/requirements/infrastructure_management/_index.en.md

```diff
@@ -144,6 +145,19 @@ The needed `output.tf` file already comes with all our
 [terraform-configs-github]: https://github.com/kubermatic/kubeone/tree/master/examples/terraform
 {{% /notice %}}
 
+## VMware
```

Comment:
Let's put this above `Terraform Integration. Also, I would put a subheading that goes something like `Cloud Provider-specific Requirements`, and then `VMware vSphere` in that subheading.

```suggestion
## Cloud Provider-specific Requirements

### VMware vSphere
```